examples/streaming/wordcount-java/src/main/java/org/apache/gearpump/streaming/examples/wordcountjava/dsl/WordCount.java

```diff
@@ -25,12 +25,17 @@
 import org.apache.gearpump.cluster.client.ClientContext;
 import org.apache.gearpump.streaming.dsl.javaapi.JavaStream;
 import org.apache.gearpump.streaming.dsl.javaapi.JavaStreamApp;
+import org.apache.gearpump.streaming.dsl.api.functions.MapFunction;
```

Comment:
dsl packages are restructured into `dsl.api` for common interfaces, `dsl.scalaapi` for Scala and `dsl.javaapi` for Java.